### DISCUSSION TITLE
码表权重数据来源

### DISCUSSION THREAD

**Discussion by Eliot00** [siuze/ShanRenMaLTS#9] [疑惑] [answered]
请问作者码表权重是怎么统计的？为什么用的浮点数？

**Comment by siuze** [comment, marked as answer]
码表的权重是依据jieba分词项目的词库来编排的，选择jieba的主要原因是其词库即带有单字又带有词组，且都有权重，可以优化候选项的排序效果，过去的码表几乎只有单字权重，词汇权重很少，字词同在候选时的排序效果不佳。

浮点数只是因为处理数据的时候忘记调整了，rime在编译部署的时候可能会自动强制转换。

但是生成单字快速编码的依据并不是jieba词库，这个我是要写一篇wiki的。但是近来在忙别的项目，估计过两周可以写出来。
